crates/fuel-gas-price-algorithm/gas-price-analysis/src/simulation.rs

```diff
@@ -207,7 +213,7 @@ impl Simulator {
                     }
                 },
             );
-        da_blocks
+        nones.chain(da_blocks).collect()
```

Comment:
```suggestion
        l2_blocks_with_no_da_blocks.chain(da_blocks).collect()
```

Reply:
Updated in https://github.com/FuelLabs/fuel-core/pull/2240/commits/ff0db6e70e8174fceb5484d13bb9e5ae9a917f68